corehq/util/datadog/utils.py

```diff
@@ -13,6 +13,26 @@
 datadog_metric_logger = logging.getLogger('datadog-metrics')
 
 WILDCARD = '*'
+DATADOG_USER_MAX_FORMS_LIMIT = 300
```

Comment:
I don't think this should be a constant here since this is a hubspot threshold: https://github.com/dimagi/commcare-hq/pull/13916/files#diff-f91ad376200614f22e56c169ab86b9a7R398